src/ScheduledJobs/Contracts/IS3Service.cs

```diff
@@ -13,5 +14,6 @@ public interface IS3Service
         Task<IEnumerable<string>> GetFilesNamesAsync(string bucketName, string prefix);
         Task SaveStringContentAsync(string bucketName, string key, string content);
         Task DeleteFilesAsync(string bucketName, IEnumerable<string> fileNames);
+        Task SaveStreamContentAsync(string s3BucketName, string fileName, MemoryStream ms);
```

Comment:
Keep it a generic `Stream` for flexibility. It doesn't have to specifically be a `MemoryStream`